kanso.json

```diff
@@ -48,8 +48,7 @@
         "app-settings": ">=0.0.2",
         "url": null,
         "querystring": null,
-        "duality": null,
-        "kanso-gardener": null
```

Comment:
oops good catch
I should really fix it for good https://github.com/medic/medic-webapp/issues/2678